testutil/fstest/file.go

```diff
@@ -56,6 +56,13 @@ func CreateDir(name string, perm os.FileMode) Applier {
 	})
 }
 
+// RemoveDir returns a file applier which removes the provided directory name
+func RemoveDir(name string) Applier {
```

Comment:
Wondering if I should have just named `RemoveFile` as `RemoveAll` instead. We shouldn't have functions that are equivalent like this. Let's just rename `RemoveFile` to `Remove` and use that.